internal/storage/storage_file.go

```diff
@@ -1,17 +1,20 @@
 package storage
 
 import (
+	"context"
 	"encoding/json"
 	"os"
 	"sync"
+
+	"github.com/google/uuid"
 )
 
 type StorageFile struct {
 	mu       sync.Mutex
 	filePath string
 }
 
-func (fs *StorageFile) SaveLongURL(longURL string) (shortURL string, err error) {
+func (fs *StorageFile) SaveLongURL(_ctx context.Context, longURL, userID string) (shortURL string, err error) {
```

Comment:
```suggestion
func (fs *StorageFile) SaveLongURL(_ context.Context, longURL, userID string) (shortURL string, err error) {
```

и далее по коду. Компилятор не будет ругаться на неиспользуемые переменные, но это считается плохой практикой, и многие линтеры такой код не пропускают.